### PR TITLE
Add recipe for virema 0.6

### PR DESCRIPTION
**Pull request by npavlovikj**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/virema/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+source:
+  fn: ViReMa_0.6.zip
+  url: https://sourceforge.net/projects/virema/files/ViReMa_0.6/ViReMa_0.6.zip
+  md5: 2e15fcea133a64b4b043dba39d87f289
+  patches:
+    - path.patch
+
+build:
+  number: 0
+  skip: True # [osx or py3k]
```
Comment (bgruening) [inline recipes/virema/meta.yaml]: Why is this skiped for OSX?

File: recipes/virema/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package:
+  name: virema
+  version: "0.6"
+
+source:
+  fn: ViReMa_0.6.zip
+  url: https://sourceforge.net/projects/virema/files/ViReMa_0.6/ViReMa_0.6.zip
+  md5: 2e15fcea133a64b4b043dba39d87f289
+  patches:
+    - path.patch
```
Comment (bgruening) [inline recipes/virema/meta.yaml]: What is this patch doing?

Comment (npavlovikj) [reply recipes/virema/meta.yaml]: @bgruening , the scripts are python scripts, so this patch adds "/opt/anaconda1anaconda2anaconda3/bin/python", and there is also "^M" in the python script after this (:?), so I remove that as well. Is that ok ?

Comment (bgruening) [reply recipes/virema/meta.yaml]: You should add `#!/usr/bin/env python` and not the long /opt/anaconda prefix. I would not touch the line endings if not needed.

Comment (npavlovikj) [reply recipes/virema/meta.yaml]: Thanks for the suggestion @bgruening , I fixed that in the newest commit.

Comment (bgruening) [reply recipes/virema/meta.yaml]: Can you try to make the patch smaller and not change line endings. Its hard to read the patch this way.

Comment (npavlovikj) [reply recipes/virema/meta.yaml]: Sorry about that @bgruening , it is fixed now.

**Comment by epruesse** [comment]
LGTM. Thanks @npavlovikj!

**Pull request merged**